README.md

```diff
@@ -8,10 +8,33 @@
 **Lawnicons** is an icon pack developed by the Lawnchair team and supported by our community.
 Originally an addon for Lawnchair 12 Alpha 5 and above to implement themed icons, it can now be used on many launchers.
 
-Lawnicons is best used [on Lawnchair 12.1 or Lawnchair 14 Beta 2](https://github.com/LawnchairLauncher/lawnchair/releases). You can enable themed icons on Lawnchair by going to `Home Settings → General → Icon Style → Themed Icons → "Home screen" or "Home screen and App Drawer"`.
+Lawnicons is best used [on the latest version of Lawnchair](https://github.com/LawnchairLauncher/lawnchair/releases). You can enable themed icons on Lawnchair by going to `Home Settings → General → Icon Style → Themed Icons → "Home screen" or "Home screen and App Drawer"`.
 
 ## Download
-See [the Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) for the latest stable build. For development builds with new icons, go to [nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
+
+### Stable Builds
+
+<p align="center">
+  <a href="https://play.google.com/store/apps/details?id=app.lawnchair.lawnicons.play">
+    <img src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png" alt="Get it on Google Play" height="76" />
+  </a>
+  <a href="https://apt.izzysoft.de/fdroid/index/apk/app.lawnchair.lawnicons">
+    <img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png" alt="Get it on IzzyOnDroid" height="76" />
+  </a>
+  <a href="https://apps.obtainium.imranr.dev/redirect?r=obtainium://add/https://github.com/LawnchairLauncher/lawnicons/">
+    <img src="https://raw.githubusercontent.com/ImranR98/Obtainium/main/assets/graphics/badge_obtainium.png" alt="Get it on Obtainium" height="54" />
+  </a>
+  <a href="https://github.com/LawnchairLauncher/lawnicons/releases/latest">
+    <img src="https://user-images.githubusercontent.com/69304392/148696068-0cfea65d-b18f-4685-82b5-329a330b1c0d.png" alt="Get it on GitHub" height="80" />
+  </a>
+</p>
+
+*Note:* Due to different package names, Lawnicons on the Play Store will install as a different app from other sources.
```

Comment:
I guess we could just italicize the full text, while adding the dev build note:

```markdown
(links)

**Development builds:** X • Y

_Note: ..._
```

Reply:
![Screenshot 2024-09-18 at 13 09 38](https://github.com/user-attachments/assets/125db4d3-b862-46e3-9571-297801305d29)
This is the neatest option if you insist that there is a need for the note.

Reply:
I've updated it to follow your suggestion. However, I think it is also important to note that most people will not care about how the README is organised, so in the end, it will probably not make much difference.

Reply:
> most people will not care about how the README is organised, as long as the links to the downloads are there, so in the end, it will probably not make much difference.

The point is to keep the description short until it is absolutely necessary to enlarge it. This makes it more accessible.

The visual cleanliness and consistency of styles feels like you're coming into the home of someone who keeps it clean. It's a small thing, as long as it's in 1-2 places, but if you start treating everything formally, you get the opposite feeling — like you've come to people who haven't cleaned their house in months.